packages/sources/deribit/src/config.ts

```diff
@@ -0,0 +1,12 @@
+import { Requester } from '@chainlink/ea-bootstrap'
+import { Config } from '@chainlink/types'
+
+export const NAME = 'DERIBIT'
+
+export const DEFAULT_BASE_URL = 'https://www.deribit.com/api/v2/public/get_historical_volatility'
```

Comment:
Same as above comment about keeping the base URL generalized.

`/get_historical_volatility` should be the `url` of `src/adapter.ts`.